### PR TITLE
Fix VCF error message

### LINKED ISSUE #3008: LoadVCF should error on call fields that are not Number=1
Instead of throwing match errors when trying to parse format fields.

### PR DESCRIPTION
**Pull request by tpoterba**
fixes #3008

### CODE REVIEW

File: src/main/scala/is/hail/io/vcf/LoadVCF.scala [hunk 1]
```diff
@@ -622,7 +622,7 @@ object LoadVCF {
       case (VCFHeaderLineType.String, false) => TString()
       case (VCFHeaderLineType.Character, false) => TString()
       case (VCFHeaderLineType.Flag, false) => TBoolean()
-      case (_, true) => fatal(s"Can only convert a header line with type `String' to a Call Type. Found `${ line.getType }'.")
+      case (_, true) => fatal(s"Can only convert a header line with type `String' to a call type. Found `${ line.getType }'.")
```
Comment (jbloom22) [inline src/main/scala/is/hail/io/vcf/LoadVCF.scala]: Type shouldn't be capitalized. How about:
Header line must have type `String' in order to cast to type Call. Found ...

File: src/main/scala/is/hail/io/vcf/LoadVCF.scala [hunk 2]
```diff
@@ -633,6 +633,8 @@ object LoadVCF {
       (line.getCount == 1 ||
         (line.getType == VCFHeaderLineType.Flag && line.getCount == 0)))
       (Field(id, baseType, i), (id, attrs))
+    else if (baseType.isInstanceOf[TCall])
+      fatal("Hail does not support call fields where 'Number' is not 1.")
```
Comment (jbloom22) [inline src/main/scala/is/hail/io/vcf/LoadVCF.scala]: How about:
Call fields must have 'Number' equal to 1.

File: src/main/scala/is/hail/io/vcf/LoadVCF.scala (hunk 1 above)
Comment (tpoterba) [reply src/main/scala/is/hail/io/vcf/LoadVCF.scala]: the String refers to the VCF type, which is capitalized

Comment (jbloom22) [reply src/main/scala/is/hail/io/vcf/LoadVCF.scala]: I meant the word "Type"

Comment (tpoterba) [reply src/main/scala/is/hail/io/vcf/LoadVCF.scala]: I lower-cased Call and Type here. Not seeing what you're seeing, sorry!

Comment (jbloom22) [reply src/main/scala/is/hail/io/vcf/LoadVCF.scala]: Oops, I was looking at the old version. Nm!

**Pull request merged**